script.js

```diff
@@ -22,26 +22,17 @@ function Idea(title, body)  {
   this.currentImportance = this.importance[this.index];
 }
 
-// function reloadCards() {
-//   for (var i = 0; i < localStorage.length; i++) {
-//     var ideaObject = JSON.parse(localStorage.getItem(localStorage.key(i)));
-//     if (ideaObject.completed === false) {
-//       prependCard(ideaObject);
-//     }
-//   }
-// }
-
 function reloadCards() {
   for (var i = 0; i < localStorage.length; i++) {
```

Comment:
So did you not end up putting stuff in localStorage as an array? It's a lot harder to iterate over an object.